python/hsfs/core/transformation_function_engine.py

```diff
@@ -296,3 +312,94 @@ def populate_builtin_transformation_functions(
                 training_dataset.transformation_functions,
                 stats.feature_descriptive_statistics,
             )
+
+    def get_ready_to_use_transformation_fns(
+        self,
+        entity: Union[
+            "hsfs.feature_view.FeatureView", "training_dataset.TrainingDataset"
+        ],
+        training_dataset_version: Optional[int] = None,
+    ) -> Dict[
+        str, "hsfs.transformation_function_attached.TransformationFunctionAttached"
+    ]:
+        if self._feature_view_api is None:
+            self._feature_view_api = feature_view_api.FeatureViewApi(
+                self._feature_store_id
+            )
+        if self._statistics_engine is None:
+            self._statistics_engine = statistics_engine.StatisticsEngine(
+                self._feature_store_id
```

Comment:
entity argument is missing here
TypeError: StatisticsEngine.__init__() missing 1 required positional argument: 'entity_type'